src/index.html

```diff
@@ -22,6 +22,428 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="image"
+        href="#"
+      >
```

Comment:
Need to fix code style everywhere if you have only 2 attributes
```suggestion
      <a class="header__logo" href="#">
```